packages/docusaurus-plugin-pwa/src/sw.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* eslint-disable no-restricted-globals */
```

Comment:
Do you still need this, if you pass the eslint environment in our config? https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
actually it looks like i needed to use `window.self` instead of `self` 😄